ad_api/base/client.py

```diff
@@ -237,7 +237,7 @@ def _request(
             method,
             self.endpoint + path,
             params=params,
-            data=request_data,
+            json=request_data,
```

Comment:
If I understand the code correctly, in this case, "data" and "json" are not interchangeable because they have different behaviors when passing a string as a value.

Have you tested your patch? I'm concerned that this change in the base client could potentially break the library.